samples/Common.c

```diff
@@ -409,13 +409,12 @@ STATUS initializePeerConnection(PSampleConfiguration pSampleConfiguration, PRtcP
     configuration.kvsRtcConfiguration.iceSetInterfaceFilterFunc = NULL;
 
     // disable TWCC
-    configuration.kvsRtcConfiguration.disableSenderSideBandwidthEstimation = TRUE;
+    configuration.kvsRtcConfiguration.disableSenderSideBandwidthEstimation = pSampleConfiguration->disableTwcc;
```

Comment:
Having `!` here is confusing due to double negative, can we make the sample configuration positive. This variable is always used inverted.

We can keep "disableSenderSideBandwidthEstimation" to avoid changing existing. But since `disableTwcc` is a new property in the sample configuration, we can invert it for easier to read.